Code/src/main/java/nl/utwente/viskell/ui/serialize/ViskellFormat.java

```diff
@@ -0,0 +1,42 @@
+package nl.utwente.viskell.ui.serialize;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by andrew on 03/04/16.
```

Comment:
We don't do these kind of file headers elsewhere - just as easy to pull this information from `git blame`.

I'll make a PR to add your name to LICENSE, though.